apis/contour/v1beta1/register.go

```diff
@@ -34,12 +34,30 @@ func Resource(resource string) schema.GroupResource {
 	return SchemeGroupVersion.WithResource(resource).GroupResource()
 }
 
-func AddKnownTypes(scheme *runtime.Scheme) {
+// AddKnownTypes is exported for backwards compatibility with third
+// parties who depend on this symbol, but all new code should use
+// AddToScheme.
+func AddKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
 		&IngressRoute{},
 		&IngressRouteList{},
 		&TLSCertificateDelegation{},
 		&TLSCertificateDelegationList{},
 	)
 	metav1.AddToGroupVersion(scheme, SchemeGroupVersion)
+	return nil
 }
+
+// The following declarations are kubebuilder-compatible and will be expected
+// by third parties who import the Contour API types.
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1beta1"}
```

Comment:
Same as previous comment, duplicated field.

Reply:
Leaving this for compatibility for now.